File: JavaNIDE/Javaprgm/app/src/main/java/sample/Forloop4.java
package sample;
import java.util.*;

 class Forloop4
 {
   public static void main(String [] ar)
   {
     int n,i,j,sum=0,prod=1,k=1;
     System.out.println("enter no of term");
     Scanner m=new Scanner (System.in);
     n=m.nextInt();
     for(i=1;i<=n;i++)
     {
      for(j=1;j<=i;j++)
      {
       System.out.print(k);
       if(j<i)
       {
         System.out.print("*");
       }
      prod=prod*k++;;
      }
      System.out.print(" + ");
     sum=sum+prod;
     prod=1;
   }
   System.out.println("\n sum of ser="+sum);
  
  }
}

File: python programming/forloop_4.py
n=int(input("enter no of term:-"))
sum=0
for i in range(1,n+1):
 print(f'{i}/[{i}-({i}/{i+1}^2)]',end='')
 if i<n:
  print("  +  ",end='')
  m=i/((i+1)*(i+1))
 sum=sum+i/(i-m)
print("sum of seriese=",sum)
 